docs/sdk/tutorials/export_a_kili_project.md

```diff
@@ -1,22 +1,28 @@
 # How to export a Kili project
 ## Outline
-This tutorial explains the multiple ways to export a Kili project. It describes a per-label method involving label filtering and conversion, and also the solutions performing a full-project export. The methods are illustrated with code snippets.
+This tutorial explains the multiple ways to export a Kili project. It describes:
+
+ * A per-label method involving label filtering and conversion
```

Comment:
the first bullet point is not clear to me. Conversion to what?

Reply:
what about: 
` * Methods to export the labels one by one after filtering` ?

Reply:
Added another bullet point. Hope it's ok now?